docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -751,3 +751,51 @@ spec:
   tags: compliance, hardening, built-in
   platform: darwin
   contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Operating system up to date (macOS)
+  query: SELECT 1 FROM os_version WHERE version >= '12.5.1';
+  description: "Checks that the operating system is up to date."
+  resolution: "Go to the Software Update section of System Preferences and install the latest updates."
```

```suggestion
  resolution: "From the Apple menu () in the corner of your screen choose System Preferences. Then select Software Update and select Upgrade Now. You might be asked to restart or enter your password."
```

I propose updating the resolution so that almost anyone with a Mac can follow the steps. 

I also suggest warning the user that they may be asked to restart or enter their password. This way, the user is aware of all steps required to turn the policy green (pass).

---

For those that don't require MDM this makes a lot of sense. Committing suggestion!